pkg/controller/s3/bucket.go

```diff
@@ -132,6 +135,9 @@ func (e *external) Observe(ctx context.Context, mg resource.Managed) (managed.Ex
 	return managed.ExternalObservation{
 		ResourceExists:   true,
 		ResourceUpToDate: true,
+		ConnectionDetails: map[string][]byte{
+			runtimev1alpha1.ResourceCredentialsSecretEndpointKey: []byte(meta.GetExternalName(cr)),
```

Comment:
Is bucket name enough here for the apps to connect?

Reply:
I can add the region as well

Reply:
Maybe the ARN would make sense?

Reply:
For connecting to an s3 bucket, only the region and bucket name is required. Ex., https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html#access-bucket-intro